.github/workflows/pr_iac_lint.yml

```diff
@@ -38,7 +42,7 @@ jobs:
           terraform -version
           terraform init -backend=false
           terraform validate
-          terraform plan
+          echo terraform plan # We can't do this without a credentials source
```

Comment:
This one needs IAM credentials ?! I've removed it so we do all of the validation apart from the part that connects to AWS. I think this is reasonable.

Reply:
OK, let's remove the part that needs AWS credentials, it will avoid failed builds for non-PT-members who push a PR.